java/Makefile

```diff
@@ -13,9 +13,11 @@ gen-java-apache-client: clean-java-apache-client
 	  --additional-properties=groupId=com.lancedb,artifactId=lance-catalog-apache-client,artifactVersion=$(VERSION),parentGroupId=com.lancedb,parentArtifactId=lance-catalog-root,parentVersion=$(VERSION),parentRelativePath=pom.xml,library=apache-httpclient,apiPackage=com.lancedb.lance.catalog.client.apache.api,modelPackage=com.lancedb.lance.catalog.client.apache.model,hideGenerationTimestamp=true,licenseName=Apache-2.0,licenseUrl=https://www.apache.org/licenses/LICENSE-2.0.txt
 	sed -i -e 's#<junit-version>5.10.2</junit-version>#<junit-version>5.8.2</junit-version>#g' lance-catalog-apache-client/pom.xml
 	rm -f lance-catalog-apache-client/pom.xml-e
-	./mvnw spotless:apply
 
-build-java-apache-client: gen-java-apache-client
+lint-java-apache-client: gen-java-apache-client gen-java-springboot-server
```

Comment:
has to separate lint and codegen, becuase lint in java has to be run after all codegen completes otherwise it gives module not found error